css/src/atomics/spacing.scss

```diff
@@ -21,211 +20,67 @@ $layout-sizes: (
 
 // Pattern: <cssproperty>-<value>-<screen>
 
-@each $size in $layout-sizes {
-	$sizeKey: nth($size, 1);
-	$sizeValue: nth($size, 2);
-
-	// .margin-<value>
-	.#{$marginKey}#{$separator}#{$sizeKey} {
-		margin: sizeValue($sizeKey, $sizeValue) !important;
-	}
-
-	// .padding-<value>
-	.#{$paddingKey}#{$separator}#{$sizeKey} {
-		padding: sizeValue($sizeKey, $sizeValue) !important;
-	}
-
-	// margin-inline and margin-block property isn't supported by IE, Edge, and Safari yet (as of Apr 2021)
-
-	// .margin-inline-<value>
-	.#{$marginKey}#{$separator}#{$inlineKey}#{$separator}#{$sizeKey} {
-		// Replace with margin-inline property when possible
-		margin-right: sizeValue($sizeKey, $sizeValue) !important;
-		margin-left: sizeValue($sizeKey, $sizeValue) !important;
-	}
-
-	// .padding-inline-<value>
-	.#{$paddingKey}#{$separator}#{$inlineKey}#{$separator}#{$sizeKey} {
-		// Replace with margin-inline property when possible
-		padding-right: sizeValue($sizeKey, $sizeValue) !important;
-		padding-left: sizeValue($sizeKey, $sizeValue) !important;
-	}
-
-	// .margin-block-<value>
-	.#{$marginKey}#{$separator}#{$blockKey}#{$separator}#{$sizeKey} {
-		// Replace with margin-block property when possible
-		margin-top: sizeValue($sizeKey, $sizeValue) !important;
-		margin-bottom: sizeValue($sizeKey, $sizeValue) !important;
-	}
-
-	// .padding-block-<value>
-	.#{$paddingKey}#{$separator}#{$blockKey}#{$separator}#{$sizeKey} {
-		// Replace with margin-block property when possible
-		padding-top: sizeValue($sizeKey, $sizeValue) !important;
-		padding-bottom: sizeValue($sizeKey, $sizeValue) !important;
-	}
-
-	@each $position-key, $direction in $directions {
-		// .margin-<side>-<value>
-		.#{$marginKey}#{$separator}#{$position-key}#{$separator}#{$sizeKey} {
-			margin-#{$direction}: sizeValue($sizeKey, $sizeValue) !important;
-		}
-
-		// .padding-<side>-<value>
-		.#{$paddingKey}#{$separator}#{$position-key}#{$separator}#{$sizeKey} {
-			padding-#{$direction}: sizeValue($sizeKey, $sizeValue) !important;
-		}
-	}
-}
-
-@include tablet {
+@each $property in $spacing-properties {
+	// iterates through margin, margin-inline, margin-block, etc
 	@each $size in $layout-sizes {
+		// iterates over values 'xs', 's', 'l' ... 'none'
```

Comment:
I know I placed these comments in a previous suggestion here - I had placed them there as explanation, not thinking they'd be included. Could we reduce/remove them? Perhaps only one per @each?

```
// .<property>-<value>-widescreen
@include widescreen {
	@each $property in $spacing-properties {
		@each $size in $layout-sizes {
			$sizeKey: nth($size, 1);
			$sizeValue: nth($size, 2);

			.#{$property}#{$separator}#{$sizeKey}#{$separator}widescreen {
				#{$property}: $sizeValue !important;
			}
		}
	}
}
```

Reply:
Removed